xclim/cli.py

```diff
@@ -0,0 +1,239 @@
+        params=params,
+        help=indicator.abstract,
+        short_help=indicator.long_name,
+    )
+
+
+@click.command(short_help="List indicators.")
+@click.argument("module", required=False, nargs=-1)
+@click.option(
+    "-i", "--info", is_flag=True, help="Prints more details for each indicator."
+)
+def list(module, info):
+    """List all indicators in MODULE
+
+    If MODULE is ommitted, lists everything.
+    """
+    if len(module) == 0:
+        module = "all"
+    formatter = click.HelpFormatter()
+    formatter.write_heading("Listing all available indicators for computation.")
+    for xcmod in [atmos, land, seaIce]:
+        modname = xcmod.__name__.split(".")[-1]
+        if module == "all" or modname in module:
+            with formatter.section(
+                click.style("Indicators in module ", fg="blue")
+                + click.style(f"{modname}", fg="yellow")
+            ):
+                formatter.write_dl(
+                    [
+                        (
+                            click.style(name, fg="yellow")
+                            + (
+                                (" (" + ind.var_name + ")")
+                                if ind.var_name != name
+                                else ""
+                            ),
+                            ind.long_name
+                            + (
+                                ("\n" + ind.abstract.replace("\n    ", ""))
+                                if info
+                                else ""
+                            ),
+                        )
+                        for name, ind in xcmod.__dict__.items()
+                        if isinstance(ind, xc.utils.Indicator)
+                    ]
+                )
+    click.echo(formatter.getvalue())
+
+
+@click.command()
+@click.argument("indicator", nargs=-1)
+@click.pass_context
+def info(ctx, indicator):
+    """Gives information about INDICATOR.
+
+    Same as "xclim INDICATOR --help"
+
+    INDICATOR must include its module (ex: atmos.tg_mean)
+    """
+    for indname in indicator:
+        ind = _get_indicator(indname)
+        command = _create_command(indname)
+
+        formatter = click.HelpFormatter()
+        with formatter.section(
+            click.style("Indicator", fg="blue")
+            + click.style(f" {indname}", fg="yellow")
+        ):
+            for attrs in [
+                "identifier",
+                "var_name",
+                "title",
+                "long_name",
+                "units",
+                "cell_methods",
+                "abstract",
+                "description",
+            ]:
+                formatter.write_text(
+                    click.style(f"{attrs}: ", fg="blue") + f"{getattr(ind, attrs)}"
+                )
+
+        command.format_options(ctx, formatter)
+
+        click.echo(formatter.getvalue())
+
+
+class XclimCli(click.MultiCommand):
+    def list_commands(self, ctx):
+        return ["list", "info"]
```

Comment:
Is there a reason why this is wrapped in a list? Wouldn't a tuple work the same?